Functions.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import math
+##These functions relate to the initiation of the code and mesh
```

Comment:
You should use the docstring notation from the notes for the definition of your functions comments

Reply:
This should be used for all of your function comments